config.json.example

```diff
@@ -9,5 +9,17 @@
     "block_explorer_api_url": "http://api.etherscan.io/api",
     "block_explorer_api_key": "ABCD1234",
     "block_explorer_api_request_delay": 0,
-    "base_currency": "ETH"
+    "base_currency": "ETH",
+    "currency_map": [
+        {
+            "blockchain_currency":"USDC.e",
+            "account_suffix": "USDC",
+            "beancount_commodity": "USD"
+        },
+        {
+            "blockchain_currency":"WETH.e",
+            "account_suffix": "ETH",
+            "beancount_commodity": "ETH"
+        }
+    ]
```

Comment:
Maybe we can use a more concise form? I would prefer this:
```json
"USDC.e": {"commodity": "USD", "account_suffix": "USDC"},
"WETH.e": {"commodity": "ETH", "account_suffix": "ETH"}
```

Reply:
Sure. This was my first idea, but for some reason, I have not been able to parse this syntax. I guess I have been limited of my beginner/casual python skills.
I'll try again .